ios/FluentUI/Notification/NotificationTokens.swift

```diff
@@ -0,0 +1,237 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Pre-defined styles of the notification
+@objc public enum MSFNotificationStyle: Int, CaseIterable {
+    /// Floating notification with brand colored text and background.
+    case primaryToast
+
+    /// Floating notification with neutral colored text and background.
+    case neutralToast
+
+    /// Bar notification with brand colored text and background.
+    case primaryBar
+
+    /// Bar notification with brand colored text and neutral colored background.
+    case primaryOutlineBar
+
+    /// Bar notification with neutral colored text and brackground.
+    case neutralBar
+
+    /// Floating notification with red text and background.
+    case dangerToast
+
+    ///Floating notification with yellow text and background.
+    case warningToast
+
+    var isToast: Bool {
+        switch self {
+        case .primaryToast,
+             .neutralToast,
+             .dangerToast,
+             .warningToast:
+            return true
+        case .primaryBar,
+             .primaryOutlineBar,
+             .neutralBar:
+            return false
+        }
+    }
+
+    var animationDurationForShow: TimeInterval { return isToast ? Constants.animationDurationForShowToast : Constants.animationDurationForShowBar }
+    var animationDurationForHide: TimeInterval { return Constants.animationDurationForHide }
+    var animationDampingRatio: CGFloat { return isToast ? Constants.animationDampingRatioForToast : 1 }
+
+    var needsFullWidth: Bool { return !isToast }
+    var needsSeparator: Bool { return  self == .primaryOutlineBar }
+    var supportsTitle: Bool { return isToast }
+    var supportsImage: Bool { return isToast }
+    var shouldAlwaysShowActionButton: Bool { return isToast }
+
+    private struct Constants {
+        static let animationDurationForShowToast: TimeInterval = 0.6
+        static let animationDurationForShowBar: TimeInterval = 0.3
+        static let animationDurationForHide: TimeInterval = 0.25
+        static let animationDampingRatioForToast: CGFloat = 0.5
+    }
+}
+
+/// Design token set for the `Notification` control.
+open class NotificationTokens: ControlTokens {
+    /// Defines the style of the notification.
+    public internal(set) var style: MSFNotificationStyle = .primaryToast
+
+    // MARK: - Design Tokens
+
+    /// The background color of the notification
+    open var backgroundColor: DynamicColor {
+        switch style {
+        case .primaryToast:
+            return globalTokens.brandColors[.tint40]
+        case .neutralToast:
+            return DynamicColor(light: ColorValue(0xF7F7F7),
```

Comment:
these colors not update yet?

Reply:
We have avoided updating Notification and TVC specifically because we knew they were in the works coming from Main soon. So correct, we have not updated these colors yet.